src/app/test_executive/test_executive.ml

```diff
@@ -120,7 +123,7 @@ let main inputs =
     | None ->
         [%log info] "cleaning up testnet ($reason, $error)"
           ~metadata:
-            [("reason", `String reason); ("hard_error", `String test_error_str)] ;
+            [("reason", `String reason); ("error", `String test_error_str)] ;
```

Comment:
If you are going to update this metadata key, can you also update the one from lines 117-121?

Reply:
Sure yeah, the keys matched there so didn't bother renaming it